themes/gatsby-theme-catalyst-bery/src/components/post-template.js

```diff
@@ -59,7 +59,6 @@ const PostTemplate = ({ data, previous, next }) => {
                     transition: "color 0.2s ease",
                     ":hover, :focus, :active": {
                       color: "secondary",
-                      textDecoration: "underline",
```

Comment:
I want to keep the underline here - again for a11y to make the link standout a bit more against the visual background.